src/app/views/query-runner/request/auth/Auth.tsx

```diff
@@ -61,7 +62,7 @@ export function Auth(props: any) {
             href={`https://jwt.ms#access_token=${accessToken}`}
             target='_blank' />
         </div>
-        <Label className={classes.accessToken} >{accessToken}</Label>
+        <Label className={classes.accessToken} >{permissionModeType === PERMISSION_MODE_TYPE.User ? accessToken : "Access token is unviewable in application mode for security reasons."}</Label>
```

Comment:
Should "Access token is unviewable in application mode for security reasons" be in constants folder?

Reply:
^ localization too

Reply:
b1a030f0112b85437d5b55222f9ee33c021da48f